auth/src/main/res/layout/fui_phone_layout.xml

```diff
@@ -56,16 +59,17 @@
                 android:layout_marginTop="@dimen/fui_field_padding_vert"
                 android:textColor="?android:textColorTertiary"
                 android:textIsSelectable="false"
+                app:layout_constraintStart_toStartOf="parent"
                 app:layout_constraintTop_toBottomOf="@+id/send_code" />
 
             <TextView
                 android:id="@+id/email_footer_tos_and_pp_text"
                 style="@style/FirebaseUI.PrivacyFooter"
                 app:layout_constraintTop_toBottomOf="@+id/send_sms_tos"
-                app:layout_constraintBottom_toBottomOf="parent" />
```

Comment:
Pretty sure this is a behavior change since the view will no longer be centered. I think we need to keep `app:layout_constraintBottom_toBottomOf` on top of `app:layout_constraintStart_toStartOf`.